data/partials/home.yaml

```diff
@@ -48,6 +48,10 @@ nav_sections:
         link: synthetics/
         icon: synthetics.svg
         desc: Ensure application uptime, identify regional issues, & track performance
+      - title: UX Monitoring
```

Comment:
Why does clicking on "UX Monitoring" leads me to a page titled "Real User Monitoring"? Can we stick to "Real User Monitoring" everywhere?

Reply:
```suggestion
      - title: Real User Monitoring
```